src/AppRoutes.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import {Route,Routes} from "react-router-dom";
+import { Home,  LogInPage, SignUpPage, VideoListPage } from "./Pages/index";
+
+export const AppRoutes = () => {
+  return (
+    <div>     
```

Comment:
You can remove this `div` as you not adding any styles

Reply:
done